src/Libraries/Microsoft.Extensions.AI.Abstractions/Embeddings/Embedding{T}.cs

```diff
@@ -19,4 +19,15 @@ public Embedding(ReadOnlyMemory<T> vector)
 
     /// <summary>Gets or sets the embedding vector this embedding represents.</summary>
     public ReadOnlyMemory<T> Vector { get; set; }
+
+    /// <summary>Gets or sets a value indicating whether the <see cref="Vector"/> has a unit norm.</summary>
```

Comment:
Do we need to qualify that it's unital w.r.t. the Euclidean norm or is it standard fare in the context of embeddings?

Reply:
I found a few resources online stating that in some cases it might be appropriate to use Manhattan or infinity, but I can't tell if they're niche or not.

Reply:
The phrasing as given is likely *precise* enough without qualifying that we mean a Euclid norm.

However I think it's probably too terse and technical for developers without a mathematical background to understand intuitively. I have a math degree but even so would have to go searching to really definitely confirm that "has a unit norm" means "is normalized to magnitude 1 in the obvious way" as opposed to "comes with an extra thing called a 'unit norm', whatever that is".

Maybe:

```suggestion
    /// <summary>Gets or sets a value indicating whether the embedding claims to normalized. If true, this indicates that the magnitude or norm of <see cref="Vector"/> equals 1.</summary>
```

Getting the "claim" terminology in here helps a bit to make sense of why the property is settable, which otherwise seems really strange. I know this is also explained in more length in `<remarks>`.

It's also nice to avoid implying that, if the vaue is `false`, the vector is *not* normalized (since it might still be).